recipes/titlecase/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "titlecase" %}
+{% set version = "1.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/titlecase-{{ version }}.tar.gz
+  sha256: 16e279edf085293bc9c44a68ce959c7d6cd5c653e6b5669a3a3640015cb63eb6
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - titlecase = titlecase.__init__:cmd
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - regex >=2020.4.4
+
+test:
+  imports:
+    - titlecase
+  commands:
+    - pip check
+    - titlecase --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ppannuto/python-titlecase
+  summary: Python Port of John Gruber's titlecase.pl
+  license: MIT
+  license_file: LICENSE.md
```

Comment:
Please add a comment above the `license_file` line that says:

```
# License file manually packaged. See: https://github.com/ppannuto/python-titlecase/pull/71
```

as a reminder to check if the license file is included in future releases.

Reply:
Addressed in 2beb30c1db58ed6028d0ab11c161d2c96e03e2b7.